apps/lockfile-explorer-web/src/parsing/getPackageFiles.ts

```diff
@@ -0,0 +1,47 @@
+import { IPackageJson } from '../types/IPackageJson';
+
+export const readPnpmfile = async (): Promise<string> => {
+  try {
+    const response = await fetch(`http://localhost:8091/api/pnpmfile`);
+    return await response.text();
+  } catch (e) {
+    console.error('Could not load cjs file: ', e);
+    return 'Missing CJS';
+  }
+};
+
+export const readPackageJSON = async (projectPath: string): Promise<IPackageJson | undefined> => {
```

Comment:
Coding convention is camelCase for acronyms longer than 2 characters:  `readPackageJson`